fluentd/lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/viaq_data_model_log_level_normalizer.rb

```diff
@@ -76,6 +76,9 @@ def normalize_level!(record, priority=nil)
             elsif !priority.nil?
               retlevel = PRIORITY_LEVELS[priority]
             end
+            if dig_structured && !record.dig('structured','level').nil?
```

Comment:
I think you could write this like the following so you don't have to execute dig twice
```
retLevel = record.dig('structured','level') if dig_stuctured

```

Also for consistency, I wonder if we should be also using the result to find an entry in NORMAL_LEVEL.  I think I may have missed that when implementing extract_level_from_message

Reply:
fixed, please review